src/lib/Service/SignalSlotService.php

```diff
@@ -306,6 +310,10 @@ private function generateNotificationEvents(string $action, Content $content, ?i
                 EventNotifierRequest::ITEM_ID_KEY => $content->id,
                 EventNotifierRequest::CONTENT_TYPE_ID_KEY => $content->contentInfo->contentTypeId,
                 EventNotifierRequest::LANG_KEY => $lang ?? null,
+                EventNotifierRequest::CREDENTIALS_KEY => [
+                    'login' => $clientCredentials->getCustomerId(),
+                    'password' => $clientCredentials->getLicenseKey()
```

Comment:
Is it correct ? AFAIK `EzRecommendationClientCredentials` represents credentials to eZ Personalization API instead of credentials protecting export files.